src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx

```diff
@@ -14,34 +14,93 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import storageService from 'ensemblRoot/src/services/storage-service';
+
+import React, { useState } from 'react';
+import { replace } from 'connected-react-router';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
 
 import { Topbar } from 'src/header/Header';
 
-import generalErrorImage1 from './images/general-error-1.jpg';
-import generalErrorImage2 from './images/general-error-2.jpg';
-import generalErrorImage3 from './images/general-error-3.jpg';
+import { ReactComponent as InfoIcon } from 'static/img/shared/icon_alert_circle.svg';
+import { PrimaryButton } from '../button/Button';
+import Chevron from '../chevron/Chevron';
 
 import styles from './ErrorScreen.scss';
+import { useDispatch } from 'react-redux';
 
-const GeneralErrorScreen = () => (
-  <section className={styles.generalErrorScreen}>
-    <Topbar />
-    <div className={styles.generalErrorBody}>
-      <p className={styles.generalErrorTopMessage}>
-        Sorry, something seems to have gone wrong
-      </p>
-      <div className={styles.generalErrorImages}>
-        <img src={generalErrorImage1} />
-        <img src={generalErrorImage2} />
-        <img src={generalErrorImage3} />
-      </div>
-      <div className={styles.generalErrorBottomMessage}>
-        <p>We‘re trying to fix it right now</p>
-        <p>Please try again later...</p>
+const GeneralErrorScreen = () => {
+  const dispatch = useDispatch();
+
+  const [moreOptionExpanded, setMoreOptionExpanded] = useState(false);
+
+  const toggleChevron = () => {
+    setMoreOptionExpanded(!moreOptionExpanded);
+  };
+
+  const chevronDirection = () => {
+    return moreOptionExpanded ? 'up' : 'down';
+  };
+
+  const reloadPage = () => {
+    window.location.reload();
+  };
+
+  const resetPage = () => {
+    storageService.clearAll();
+    dispatch(replace(urlFor.home()));
+    reloadPage();
```

Comment:
wouldn't `window.location.href = urlFor.home()` do the work of both above lines at once?

Reply:
yes it will do, but @imransl suggested to use replace(). Wondering which is better

Reply:
`replace` is probably fine (I've never used it, so don't know); but the way the code is written here what gets called is `window.location.reload()`, which looks excessive. `window.location.replace(urlFor.home())` would probably be sufficient?